src/shared/components/RadioButton/RadioButton.style.ts

```diff
@@ -67,6 +70,9 @@ export const StyledInput = styled.div<RadioButtonStyleProps>`
   align-items: center;
   background-clip: content-box;
   padding: 4px;
+  box-sizing: border-box;
+  width: 24px;
+  height: 24px;
```

Comment:
```suggestion
  width: 18px;
  height: 18px;
```

when selecting the component on figma it looks like its 24px but its misleading as it seems to have some padding in there they actually are 18px